pkg/apps/blackduck/v1/creater.go

```diff
@@ -120,8 +119,9 @@ func (hc *Creater) Ensure(blackduck *blackduckapi.Blackduck) error {
 			return fmt.Errorf("update cfssl component: %+v", errors)
 		}
 
-		if err := util.ValidatePodsAreRunningInNamespace(hc.kubeClient, blackduck.Spec.Namespace, hc.config.PodWaitTimeoutSeconds); err != nil {
-			return err
+		err = util.WaitUntilPodsAreReady(hc.kubeClient, blackduck.Spec.Namespace, fmt.Sprintf("app=%s,name=%s,component in (configmap,serviceAccount,cfssl)", util.BlackDuckName, blackduck.Name), hc.config.PodWaitTimeoutSeconds)
```

Comment:
configmap, service account can be removed from the in clause

Reply:
Done